lib/less-parser.js

```diff
@@ -114,9 +114,14 @@ export default class LessParser extends Parser {
       // eslint-disable-next-line
       delete this.current.nodes;
 
-      if (/!important/i.test(node.selector)) {
+      if (/!\s*important/i.test(node.selector)) {
         node.important = true;
-        node.selector = node.selector.replace(/\s*!important/i, '');
+
+        if (/\s*!\s+important/i.test(node.selector)) {
+          node.raws.important = node.selector.match(/(\s*!\s+important)/i)[1];
```

Comment:
I put it here before the `node.selector` normalization because postcss includes any spaces before the `!` in `node.raws.important`.